src/char/char.cpp

```diff
@@ -296,7 +296,7 @@ int32 char_mmo_char_tosql(uint32 char_id, struct mmo_charstatus* p){
 		(p->rename != cp->rename) || (p->robe != cp->robe) || (p->character_moves != cp->character_moves) ||
 		(p->unban_time != cp->unban_time) || (p->font != cp->font) || (p->uniqueitem_counter != cp->uniqueitem_counter) ||
 		(p->hotkey_rowshift != cp->hotkey_rowshift) || (p->clan_id != cp->clan_id ) || (p->title_id != cp->title_id) ||
-		(p->show_equip != cp->show_equip) || (p->hotkey_rowshift2 != cp->hotkey_rowshift2) ||
+		(p->show_equip != cp->show_equip) || (p->disable_showcostumes != cp->disable_showcostumes) || (p->hotkey_rowshift2 != cp->hotkey_rowshift2) ||
```

Comment:
Can you please move all these to below?
`//Values that will seldom change (to speed up saving)`